tracer/src/Datadog.Trace/Telemetry/Metrics/MetricTags.cs

```diff
@@ -0,0 +1,142 @@
+
+    // ContextHeaderStyle
+    [Description("header_style:tracecontext")] ContextHeaderStyle_TraceContext,
+    [Description("header_style:datadog")] ContextHeaderStyle_Datadog,
+    [Description("header_style:b3multi")] ContextHeaderStyle_B3Multi,
+    [Description("header_style:b3single")] ContextHeaderStyle_B3SingleHeader,
+
+    // TelemetryAPI endpoint
+    [Description("endpoint:agent")] TelemetryApi_Agent,
+    [Description("endpoint:agentless")] TelemetryApi_Agentless,
+
+    // Instrumentation counts
+    [Description("component_name:trace_annotations")] InstrumentationCounts_TraceAnnotations,
+    [Description("component_name:dd_trace_methods")] InstrumentationCounts_DDTraceMethods,
+    [Description("component_name:calltarget")] InstrumentationCounts_CallTarget,
+    [Description("component_name:calltarget_derived")] InstrumentationCounts_CallTargetDerived,
+
+    // Integration name
+    [Description("integrations_name:httpmessagehandler")]IntegrationName_HttpMessageHandler,
+    [Description("integrations_name:aspnetcore")]IntegrationName_AspNetCore,
+    [Description("integrations_name:adonet")]IntegrationName_AdoNet,
+    [Description("integrations_name:aspnet")]IntegrationName_AspNet,
+    [Description("integrations_name:aspnetmvc")]IntegrationName_AspNetMvc,
+    [Description("integrations_name:aspnetwebapi2")]IntegrationName_AspNetWebApi2,
+    [Description("integrations_name:graphql")]IntegrationName_GraphQL,
+    [Description("integrations_name:hotchocolate")]IntegrationName_HotChocolate,
+    [Description("integrations_name:mongodb")]IntegrationName_MongoDb,
+    [Description("integrations_name:xunit")]IntegrationName_XUnit,
+    [Description("integrations_name:nunit")]IntegrationName_NUnit,
+    [Description("integrations_name:mstestv2")]IntegrationName_MsTestV2,
+    [Description("integrations_name:wcf")]IntegrationName_Wcf,
+    [Description("integrations_name:webrequest")]IntegrationName_WebRequest,
+    [Description("integrations_name:elasticsearchnet")]IntegrationName_ElasticsearchNet,
+    [Description("integrations_name:servicestackredis")]IntegrationName_ServiceStackRedis,
+    [Description("integrations_name:stackexchangeredis")]IntegrationName_StackExchangeRedis,
+    [Description("integrations_name:serviceremoting")]IntegrationName_ServiceRemoting,
+    [Description("integrations_name:rabbitmq")]IntegrationName_RabbitMQ,
+    [Description("integrations_name:msmq")]IntegrationName_Msmq,
+    [Description("integrations_name:kafka")]IntegrationName_Kafka,
+    [Description("integrations_name:cosmosdb")]IntegrationName_CosmosDb,
+    [Description("integrations_name:awssdk")]IntegrationName_AwsSdk,
+    [Description("integrations_name:awssqs")]IntegrationName_AwsSqs,
+    [Description("integrations_name:ilogger")]IntegrationName_ILogger,
+    [Description("integrations_name:aerospike")]IntegrationName_Aerospike,
+    [Description("integrations_name:azurefunctions")]IntegrationName_AzureFunctions,
```

Comment:
Yeah - interestingly I missed that because there's no `IntegrationId` for it... I wonder if there _should_ be 🤔 It would seem strange to disable the Lambda integration when running in Lambda though...